src/styles/_variables.scss

```diff
@@ -4,13 +4,22 @@ $mainFont: "Montserrat", sans-serif;
 
 $black: #000;
 $white: #fff;
-$gray: #F2F2F2;
+$gray: #f2f2f2;
 $lightThemeBackground: #fff;
-$lightThemeAccept: #6C488A; 
+$lightThemeAccept: #6c488a;
 $lightThemeHeader: #696969;
 
-$darkThemeBackground: #180C30;
+$darkThemeBackground: #180c30;
+
+//screens
+$mediaPcScreen: 1180px;
+$mediaFirstTabletScreen: 991px;
+$mediaSecondTabletScreen: 767px;
+$mediaMobileScreen: 575px;
+$widthForContentPcScreen: 930px;
+$widthForContentFirstTabletScreen: 730px;
+$widthForContentSecondTabletScreen: 560px;
 
 $transparentBlackBackground: rgba(0, 0, 0, 0.33);
```

Comment:
drop it and replace to `opacity: 33%`

Reply:
Are u sure about this? If I change to opacity me content will have opacity to, now I use this variable for make background.